s4e-web/src/app/views/settings/dashboard/dashboard.component.ts

```diff
@@ -1,15 +1,15 @@
-import { Component, OnInit } from '@angular/core';
+import { ProfileQuery } from './../../../state/profile/profile.query';
+import { Component } from '@angular/core';
 
 @Component({
   selector: 's4e-dashboard',
   templateUrl: './dashboard.component.html',
   styleUrls: ['./dashboard.component.scss']
 })
-export class DashboardComponent implements OnInit {
+export class DashboardComponent {
+  constructor(private _profileQuery: ProfileQuery) {}
 
-  constructor() { }
-
-  ngOnInit() {
+  get isAdmin() {
+    return this._profileQuery.isAdmin$();
   }
```

Comment:
calling method which returns observable in a getter of the component is a very bad idea, which may lead to a lot of hard to detect bugs. Is you need `isAdmin` observable in the component you should assign it in the `ngOnInit`